bot/discord/commands/votemute.js

```diff
@@ -0,0 +1,54 @@
+const ms = require('ms');
+const Discord = require('discord.js');
+const mutesData = new require('quick.db').table("muteData");
+
+exports.run = async(client, message, args) => {
+  let modlog = message.guild.channels.cache.find(channel => channel.id === config.DiscordBot.modLogs);
+  if (!message.member.roles.cache.find(r => r.id === "898041754564046869")) return message.reply("Sorry, but it looks like you're too much of a boomer to run this command.");
+  if (args.length < 2) return message.channel.send('', { embed: new Discord.MessageEmbed().setColor(0x00A2E8).setDescription(`Correct usage ${config.DiscordBot.Prefix}votemute <@user|userID> [Time : 5m] [Reason : unspecified]`).setFooter('<required> [optional]')});
+  const allowed_channels = ["898041849783148585", "898041865616650240"];
+  if (allowed_channels.indexOf(client.channels.cache.get('id')) == -1) return; // Only lounge and dono lounge
+  let target = message.guild.members.cache.get(args[0].match(/[0-9]{18}/).length == 0 ? args[0] : args[0].match(/[0-9]{18}/)[0])
+  let reason = args.slice(2).join(' ');
+  let time = ms(args[1]) || 300000;
+  
+  if (!target) return message.reply("Please specify a valid user.");
+  if (!reason) return message.reply("Please specify a valid reason.");
+  if (time > 7200000) time = 7200000; // If the time value is over 2 hours
+  if (time < 5000) time = 5000; // If the time value is less than 5 seconds
+
+  const yes = '👍';
```

Comment:
No point of this. Just put the fucking emoji and not have a random var for the emoji